c8run/internal/unix/unix.go

```diff
@@ -0,0 +1,64 @@
+//go:build linux || darwin
+
+package unix
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"syscall"
+)
+
+func (w *UnixC8Run) OpenBrowser(name string) error {
+	operateUrl := "http://localhost:8080/operate/login"
+	var openBrowserCmdString string
+	if runtime.GOOS == "darwin" {
+		openBrowserCmdString = "open"
+	} else if runtime.GOOS == "linux" {
+		openBrowserCmdString = "xdg-open"
+	} else {
+		return fmt.Errorf("platform %s is not supported", runtime.GOOS)
+	}
+	openBrowserCmd := exec.Command(openBrowserCmdString, operateUrl)
+	fmt.Println(name + " has successfully been started.")
+	openBrowserCmd.Run()
+	return nil
+}
+
+func (w *UnixC8Run) GetProcessTree(commandPid int) []*os.Process {
+	// For unix systems we can kill all processes within a process group by setting a pgid
+	// therefore, only the main process needs put in here.
+	process := os.Process{Pid: commandPid}
+	processes := []*os.Process{&process}
+	return processes
+}
+
+func (w *UnixC8Run) GetVersionCmd(javaBinaryPath string) *exec.Cmd {
+	return exec.Command(javaBinaryPath, "--version")
+}
+
+func (w *UnixC8Run) GetElasticsearchCmd(elasticsearchVersion string, parentDir string) *exec.Cmd {
+	elasticsearchCmdString := filepath.Join(parentDir, "elasticsearch-"+elasticsearchVersion, "bin", "elasticsearch")
+	elasticsearchCmd := exec.Command(elasticsearchCmdString, "-E", "xpack.ml.enabled=false", "-E", "xpack.security.enabled=false")
+	elasticsearchCmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
+	return elasticsearchCmd
+}
+
+func (w *UnixC8Run) GetConnectorsCmd(javaBinary string, parentDir string, camundaVersion string) *exec.Cmd {
+	classPath := parentDir + "/*:" + parentDir + "/custom_connectors/*:" + parentDir + "/camunda-zeebe-" + camundaVersion + "/lib/*"
+	mainClass := "io.camunda.connector.runtime.app.ConnectorRuntimeApplication"
+	springConfigLocation := "--spring.config.location=" + parentDir + "/connectors-application.properties"
+	connectorsCmd := exec.Command(javaBinary, "-cp", classPath, mainClass, springConfigLocation)
+	connectorsCmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
+	return connectorsCmd
+}
+
+func (w *UnixC8Run) GetCamundaCmd(camundaVersion string, parentDir string, extraArgs string) *exec.Cmd {
+	camundaCmdString := parentDir + "/camunda-zeebe-" + camundaVersion + "/bin/camunda"
+	fmt.Println(camundaCmdString)
```

Comment:
I would leave the printing of the command outside of this function to have a cleaner separation of concerns.

Reply:
Good point. I added 377cc498872 to move the print statements into the main function (or remove them in cases where it wasn't useful.